cookbook/testing.md

```diff
@@ -1,30 +1,27 @@
-# Aún no está traducido...
----
+# Almacenes de prueba
 
-# Testing stores
+Los almacenes, por su diseño, se utilizarán en muchos lugares y pueden hacer que las pruebas sean mucho más difíciles de lo que deberían. Afortunadamente, esto no tiene por qué ser así. Tenemos tener cuidado de tres cosas al probar almacenes:
```

Comment:
"Tenemos **que** tener cuidado de..." instead of "Tenemos tener cuidado de".

Reply:
done